### DISCUSSION TITLE
Budibase front-end freezing after row saved from form. (Self host, V 2.29.29, Mysql DB datasource)

### DISCUSSION THREAD

**Discussion by LifeofGlen** [Budibase/budibase#14434] [Help]
Hi all,

I have a screen where a user can complete a form and then it saves a new row into a mysql table as shown below:
![image](https://github.com/user-attachments/assets/34c93ab0-cf4b-42be-b92c-ee9643d02aef)

After the button is clicked, the modal is closed and i get the row saved message:
![image](https://github.com/user-attachments/assets/06a08f70-5e40-4b09-a1a8-60b9b708dfbb)

if you try and click anywhere though, you get the following:
![image](https://github.com/user-attachments/assets/cb3d7afa-5463-4c2e-bf93-622029f59c5b)

I have to click exit page and then refresh for it to reload, but when it does, it works normal again.

The row is always inserted and after exit page and reload, life goes on.

The really strange thing, is that I have the EXACT same form (that i copied from another screen) that inserts into the same table and when I use it on that screen, it works fine.

The button only does 3 things, this is on the page that works:
![image](https://github.com/user-attachments/assets/2f30e06b-9f02-494a-81ab-b0e94f041ce7)
![image](https://github.com/user-attachments/assets/914d8fcc-72d2-4155-952d-8f7aa3bd98e3)

This is on the one that doesnt:
![image](https://github.com/user-attachments/assets/5caad378-1fb6-43b5-b923-0e2ffc4bbbd5)
![image](https://github.com/user-attachments/assets/5d3d64e6-bbdf-4759-99c6-98d498f204cc)

As I mentioned, all the rows are saved though in the DB perfectly fine (excuse my increasingly manic notes column):
![image](https://github.com/user-attachments/assets/34ed52b3-e2dc-4045-847d-db7b09320e1a)

Is there anyway to debug this further? I have looked at the logs but they don't seem related and all of them show 200 response statuses anyways:
![image](https://github.com/user-attachments/assets/4b320222-fdb1-41cf-bbe5-38e0e1027177)

Any help will be appreciated. This is a POC app that will hopefully allow us to strategically choose Budibase for all internal tools, so understanding why this could happen & how to figure/debug/fix these things is greatly appreciated.

**Comment by mikesealey** [comment]
Hey @LifeofGlen I'm sorry to hear you're having this issue. the most recent version of Budibase is a little bit ahead of you at 2.31.1, so updating would be a good idea.

Out of interest - does this behaviour happen in the preview (either the preview-modal or in the _open-in-new-tab_ preview)?

**Reply by LifeofGlen** [reply]
Hey Mike,

It happens in the preview, preview full screen and the published version.

I am busy updating now to latest version.

As further information:
I have about 8 of these screens, that all insert into the same table, all copied from an original and on about half it works perfect and half it works to insert, but then does the freeze thing on the front end.

I restarted the docker and restarted the virtual EC2 for testing and the issue still persisted, so i think it has to be something with setup or config in busibase app that i'm missing. I also tested inserting directly in mysql DB without issues and as i mentioned, the insert does always occur even through Budi when it freezes. One final note, i have been heavily adjusting and updating this app as im testing and learning new things. Ive recreated and reloaded queries, tables, relationships etc multiple times, so perhaps something is conflicting somewhere in the backend, but i am unable to identify/test.

Will follow up after updating to latest release and testing again.

Thanks for the help :)

**Reply by mikesealey** [reply]
Hey @LifeofGlen 

I've talked with a colleague and we can't see anything immediately obvious, so we'd like to take a closer look, if we may. Please could you email an app-export to support@budibase.com with using the title "Github Discussion 14434", along with any additional information you feel might be useful or important.
